docs/modules/ROOT/pages/references/quality-requirements/reliability/portal-api-data-recoverability.adoc

```diff
@@ -0,0 +1,26 @@
+= Data compatibility with external components
+
+Source::
+{product} administrator
+
+Stimulus::
+Reinstall APPUiO Global from scratch
+
+Environment::
+{zone}
+
+Artifact::
+Portal API, external third-party component
+
+Response::
+Entities that have their lifecycle outside of {product} have their availability restored
+
+Response measure::
+Following entities are recovered at latest after 48 hours since reinstallation:
```

Comment:
```suggestion
Following entities are recovered at latest after 8 hours since reinstallation:
```

I feel 48 hours is way too long, it should be possible to restore much faster.